editor/question-ctrl.js

```diff
@@ -375,6 +376,7 @@ angular.module('org.ekstep.question', ['org.ekstep.metadataform'])
   	$scope.questionData.isPartialScore = questionData1.data.config.partial_scoring;
   	$scope.questionData.gradeLevel = questionData1.data.config.metadata.gradeLevel;
   	$scope.questionData.isShuffleOption = questionData1.data.config.isShuffleOption;
+  	$scope.questionData.eval_unordered = questionData1.data.config.eval_unordered;
```

Comment:
Use lowerCamelCase for variable declaration for consistency. (eval_unordered  => evalUnordered)